lib/util/npm-util.js

```diff
@@ -53,7 +53,7 @@ function installSyncSaveDev(packages) {
     if (Array.isArray(packages)) {
         packages = packages.join(" ");
     }
-    shell.exec(`npm i --save-dev ${packages}`, { stdio: "inherit" });
+    childProcess.execSync(`npm i --save-dev ${packages}`, { stdio: "inherit", encoding: "utf8" });
```

Comment:
I tested it in Windows (cmd/ps) and Windows 10 Bash both work as expected.

It is `childProccess.spawn()` that has problems with running batch files but `childProcess.exe()` runs in a command prompt or shell that allows it to run normally.